crates/iota-sdk/examples/governance_api.rs

```diff
@@ -41,23 +40,19 @@ async fn main() -> Result<(), anyhow::Error> {
     println!("{:?}", iota_system_state);
     println!(" *** IOTA System State ***\n");
 
-    // List all active validators
+    // List all committee members
 
-    let active_validators = match iota_system_state {
-        IotaSystemStateSummary::V1(v1) => v1.active_validators,
-        IotaSystemStateSummary::V2(v2) => v2.active_validators,
-        _ => panic!("unsupported IotaSystemStateSummary"),
-    };
+    let committee_members = iota_system_state.iter_committee_members();
 
-    println!(" *** List active validators *** ");
-    active_validators.into_iter().for_each(|validator| {
+    println!(" *** List committee members *** ");
+    committee_members.for_each(|validator| {
         println!(
             "Name: {}, Description: {}, IotaAddress: {:?}",
             validator.name, validator.description, validator.iota_address
         )
     });
 
-    println!(" *** List active validators ***\n");
+    println!(" *** List committee members ***\n");
```

Comment:
I would rather list all validators here and just don't mention `active`
Further above we call `get_committee_info()` which returns the validators of the committee already

Reply:
Makes sense, modified it at [34cf64a](https://github.com/iotaledger/iota/pull/6015/commits/34cf64ad91d7e698d0877b187755424c742ee284).